packages/wrangler/package.json

```diff
@@ -190,7 +190,8 @@
 		"ws": "^8.5.0",
 		"xdg-app-paths": "^7.3.0",
 		"yargs": "^17.4.1",
-		"yoga-layout": "file:../../vendor/yoga-layout-2.0.0-beta.1.tgz"
+		"yoga-layout": "file:../../vendor/yoga-layout-2.0.0-beta.1.tgz",
+		"wrangler": "workspace:wrangler"
```

Comment:
🤨

Reply:
Needed it for the e2e tests to be able to import from 'wrangler'

Reply:
wouldn't `workspace:*` work and it would link to the `wrangler` package due to the dep name? I am just confused by the `workspace:wrangler`